cmd/vcert/commands.go

```diff
@@ -145,6 +147,24 @@ var (
 		vcert getpolicy -u https://tpp.example.com -t <TPP access token> -z "<policy folder DN>"
 		vcert getpolicy -k <VaaS API key> -z "<app name>\<CIT alias>"`,
 	}
+
+	commandSshPickup = &cli.Command{
+		Before:    runBeforeCommand,
+		Name:      commandSshPickupName,
+		Flags:     sshPickupFlags,
+		Action:    doCommandSshPickup,
+		Usage:     "To retrieve a SSH Certificate",
+		UsageText: `vcert sshpickup -u https://tpp.example.com -t <TPP access token> --pickup-id <ssh cert DN>"`,
+	}
+
+	commandSshEnroll = &cli.Command{
+		Before:    runBeforeCommand,
+		Name:      commandSshEnrollName,
+		Flags:     sshEnrollFlags,
+		Action:    doCommandEnrollSshCert,
+		Usage:     "To enroll a SSH Certificate",
+		UsageText: `vcert sshenroll -u https://tpp.example.com -t <TPP access token> --template "<val>" --id <val> --principal "bob" --principal "alice" --valid-hours 1"`,
```

Comment:
Trailing double-quote should be removed.

Reply:
fixed on most recent commit